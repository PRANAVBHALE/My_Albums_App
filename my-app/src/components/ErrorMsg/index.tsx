import React from "react";

import { Typography } from 'antd';

const { Title } = Typography;

const ErrorMsg = () => {
  return (
      <div style={{
          height: "100vh",
          display: "flex",
          width: '100%',
          alignItems: 'center',
          justifyContent: 'center'
      }}>
          <Title
              level={5}
          >
              Opps!!! Something went wrong!
          </Title>
      </div>
  );
}

export default ErrorMsg