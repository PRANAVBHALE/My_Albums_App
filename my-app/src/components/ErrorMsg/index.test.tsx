import { screen, render } from "@testing-library/react";
import ErrorMsg from ".";

describe("ErrorMsg component", () => {
  it("should render and match snapshot", () => {
    let view = render(<ErrorMsg />);

    expect(view).toMatchSnapshot();
  });

  it('should display "Opps!!! Something went wrong!"', () => {
    render(<ErrorMsg />);
    let query = screen.getByText("Opps!!! Something went wrong!");

    expect(query).toBeInTheDocument();
  });
});
