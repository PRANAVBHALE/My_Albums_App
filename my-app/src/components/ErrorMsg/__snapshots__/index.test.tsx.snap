// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMsg component should render and match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        style="height: 100vh; display: flex; width: 100%; align-items: center; justify-content: center;"
      >
        <h5
          class="ant-typography css-dev-only-do-not-override-15rg2km"
        >
          Opps!!! Something went wrong!
        </h5>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      style="height: 100vh; display: flex; width: 100%; align-items: center; justify-content: center;"
    >
      <h5
        class="ant-typography css-dev-only-do-not-override-15rg2km"
      >
        Opps!!! Something went wrong!
      </h5>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
